tools/project.py

```diff
@@ -175,7 +175,7 @@ def main():
                         type=argparse_profile_filestring_type,
                         help="Build profile to use. Can be either path to json" \
                         "file or one of the default one ({})".format(", ".join(list_profiles())),
-                        default=[])
+                        default=[argparse_profile_filestring_type('debug')])
```

Comment:
I don't think this will do what you seem to think. I think it will always start with the debug profile and merge all of the profiles specified on the command line. That's not quite what we want here.

What we want here is for the default to be the debug profile.